test/e2e-common.sh

```diff
@@ -27,7 +27,7 @@ source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
 function install_pipeline_crd() {
   echo ">> Deploying Tekton Pipelines"
-  kubectl apply -f https://storage.googleapis.com/tekton-releases/latest/release.yaml || fail_test "Tekton Pipeline installation failed"
```

Comment:
This is unsupported now (although it got updated by mistake today :sweat_smile:) hence the change.
This meant that before today, triggers was running against 0.7.0 :sweat_smile: 
We are going to remove those files is the near future.

Reply:
we need to do the same for cli/catalog fyi

Reply:
oh :sweat:

Reply:
In cli, we are getting from github release. That's better.

Reply:
That's not the case I think, we are using correct one there

Reply:
Yes. Same case in the catalog.

Reply:
ah okay I did the right thing 😅  nm then thanks for checking...

Reply:
can we get the same logic on triggers too ? try nightly and if that doesnt work fallback to latest github release?

Reply:
@dibyom @vtereso @wlynch ^^
In any case, it can be a separate PR from this one :wink:

Reply:
Updated the logic as same as CLI/Catalog